src/woo_publications/publications/tests/test_publications_api.py

```diff
@@ -1096,6 +1168,61 @@ def test_update_publication(self):
 
             self.assertEqual(response_data, expected_data)
 
+    def test_update_publication_onderwerpen_field(self):
+        topic = TopicFactory.create()
+        ic = InformationCategoryFactory.create(
+            oorsprong=InformationCategoryOrigins.value_list
+        )
+        organisation = OrganisationFactory.create(is_actief=True)
+        publication = PublicationFactory.create(
+            informatie_categorieen=[ic],
+            onderwerpen=[topic],
+            publisher=organisation,
+            publicatiestatus=PublicationStatusOptions.concept,
+            officiele_titel="title one",
+            verkorte_titel="one",
+            omschrijving="Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        )
+        detail_url = reverse(
+            "api:publication-detail",
+            kwargs={"uuid": str(publication.uuid)},
+        )
+
+        with self.subTest("update without onderwerpen doesn't alter the value"):
+            data = {
+                "informatieCategorieen": [str(ic.uuid)],
+                "publisher": str(organisation.uuid),
+                "verantwoordelijke": str(organisation.uuid),
+                "opsteller": str(organisation.uuid),
+                "publicatiestatus": PublicationStatusOptions.published,
+                "officieleTitel": "changed offical title",
+                "verkorteTitel": "changed short title",
+                "omschrijving": "changed description",
+            }
+
+            response = self.client.put(detail_url, data, headers=AUDIT_HEADERS)
+
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            self.assertEqual(response.json()["onderwerpen"], [str(topic.uuid)])
+
+        with self.subTest("update onderwerpen with empty list"):
+            data = {
+                "informatieCategorieen": [str(ic.uuid)],
+                "onderwerpen": [],  # relevant field
+                "publisher": str(organisation.uuid),
+                "verantwoordelijke": str(organisation.uuid),
+                "opsteller": str(organisation.uuid),
+                "publicatiestatus": PublicationStatusOptions.published,
+                "officieleTitel": "changed offical title",
+                "verkorteTitel": "changed short title",
+                "omschrijving": "changed description",
+            }
+
+            response = self.client.put(detail_url, data, headers=AUDIT_HEADERS)
+
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            self.assertEqual(response.json()["onderwerpen"], [])
```

Comment:
can you assert on the model state too and not just the API response?

I've seen things where the serializer outputs different things than what's in the database :)